### PR TITLE
Introduce the aarch64 support

### PR DESCRIPTION
**Pull request by LKHN**
Add arrch64 iso source

Signed-off-by: Elkhan Mammadli <elkhan.mammadli@protonmail.com>

### CODE REVIEW

File: variables.pkr.hcl [hunk 1]
```diff
@@ -2,8 +2,10 @@ variables {
   //
   // common variables
   //
-  iso_url               = "http://repo.almalinux.org/almalinux/8.4/isos/x86_64/AlmaLinux-8.4-x86_64-boot.iso"
-  iso_checksum          = "file:http://repo.almalinux.org/almalinux/8.4/isos/x86_64/CHECKSUM"
+  iso_url_x86_64        = "https://repo.almalinux.org/almalinux/8.4/isos/x86_64/AlmaLinux-8.4-x86_64-boot.iso"
+  iso_checksum_x86_64   = "file:https://repo.almalinux.org/almalinux/8.4/isos/x86_64/CHECKSUM"
+  iso_url_aarch64       = "https://repo.almalinux.org/almalinux/8.4/isos/aarch64/AlmaLinux-8.4-aarch64-boot.iso"
```
Comment (srbala) [inline variables.pkr.hcl:7]: @LKHN where does the `aarch64` variables used? 🤔

**Review by srbala — APPROVED** [review]
LGTM, is this prep for `aarch64`?

File: variables.pkr.hcl (hunk 1 above)
Comment (LKHN) [reply variables.pkr.hcl:7]: Stay tuned 😉

**Comment by LKHN** [comment]
> LGTM, is this prep for `aarch64`?

Yes

**Pull request merged**